src/memo/memo.mli

```diff
@@ -11,6 +24,8 @@ module Stack_frame : sig
 
   val name : t -> string
   val input : t -> Sexp.t
+
+  val as_instance_of : t -> of_:('input, _, _) memo -> 'input option
```

Comment:
Could you add an odoc string for this function as well?

Reply:
Added.